### PR TITLE
Increase nginx proxy timeout to support more intensive computations

### PR DESCRIPTION
**Pull request by nolim1t**
Increase NGINX proxy timeout to support cool shit like https://github.com/getumbrel/umbrel-middleware/pull/60

### CODE REVIEW

File: nginx/nginx.conf [hunk 1]
```diff
@@ -17,7 +17,7 @@ http {
 
 	access_log  /etc/nginx/log/access.log  main;
 	sendfile        on;
-	keepalive_timeout  65;
-
-	 include /etc/nginx/conf.d/*.conf;	
+	keepalive_timeout  360;
+    proxy_read_timeout 600;
+    include /etc/nginx/conf.d/*.conf;
```
Comment (jgmontoya) [inline nginx/nginx.conf:22]: Is there any rationale for those numbers or where they decided just based on trying and getting `gettxoutsetinfo` to run?

Not relevant but consistent indentation :man_shrugging: :
```suggestion
	keepalive_timeout  360;
	proxy_read_timeout 600;
	include /etc/nginx/conf.d/*.conf;	
```

**Comment by mayankchhabra** [comment]
Thanks @nolim1t. I don't really think it's a good idea as increased timeouts can have their own side-effects. Also see https://github.com/getumbrel/umbrel-middleware/pull/60#issuecomment-691425143.

File: nginx/nginx.conf (hunk 1 above)
Comment (nolim1t) [reply nginx/nginx.conf:22]: Also might fix other gateway timeouts

**Comment by nolim1t** [comment]
Might fix other gateway timeout issues too

**Comment by mayankchhabra** [comment]
Closed by https://github.com/getumbrel/umbrel-middleware/pull/60

**Pull request closed without merging**